common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -182,52 +198,15 @@ open class MusicService : MediaBrowserServiceCompat(), SessionAvailabilityListen
 
         // ExoPlayer will manage the MediaSession for us.
         mediaSessionConnector = MediaSessionConnector(mediaSession)
+        mediaSessionConnector.setPlaybackPreparer(UampPlaybackPreparer())
+        mediaSessionConnector.setQueueNavigator(UampQueueNavigator(mediaSession))
 
-        if (castPlayer.isCastSessionAvailable) {
-            mediaSessionConnector.connectToCastPlayer()
-        } else {
-            mediaSessionConnector.connectToExoPlayer(this)
-        }
-
+        switchToPlayer(/* previousPlayer= */ null, if (castPlayer.isCastSessionAvailable) castPlayer else exoPlayer)
```

Comment:
Same as above here :) Use the named arguments.

Reply:
Done! Thanks